tests/SbrpTests/ValidationTests.cs

```diff
@@ -18,71 +18,69 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace ValidateSbrpTests;
+namespace SbrpTests;
 
-public class ValidateSbrpTests
+public class ValidationTests
 {
-    private string[] Packages { get; set; }
-
     private const string SbrpAttributeType = "System.Reflection.AssemblyMetadataAttribute";
+    private const string VersionPattern = @"(\.\d)+([\-\w])*";
+    private readonly string[] Packages = Array.Empty<string>();
+    public ITestOutputHelper Output { get; set; }
 
-    private const string VerionPattern = @"(\.\d)+([\-\w])*";
-
-    private string RepoRoot { get; set; }
-
-    public ITestOutputHelper output { get; set; }
-
-    public ValidateSbrpTests(ITestOutputHelper _output)
+    public ValidationTests(ITestOutputHelper output)
     {
-        output = _output;
-
-        RepoRoot = Environment.CurrentDirectory.Substring(0, Environment.CurrentDirectory.IndexOf("artifacts"));
-
-        string buildPackagesDirectory = Path.Combine(RepoRoot, "artifacts/source-build/self/src/artifacts/packages/Release/Shipping");
+        Utilities.ValidateConfigParameters(new string[] { Config.RepoRootEnv, Config.BuildTypeEnv });
+    
+        Output = output;
+        string buildPackagesDirectory = Path.Combine(Config.RepoRoot, "artifacts/source-build/self/src/artifacts/packages", Config.BuildType, "Shipping");
 
         if (!Directory.Exists(buildPackagesDirectory))
         {
-            throw new DirectoryNotFoundException($"Directory {buildPackagesDirectory} does not exist. Try building with './build.sh -sb' first.");
+            throw new DirectoryNotFoundException($"Directory {buildPackagesDirectory} does not exist, try building with './build.sh'.");
```

Comment:
This repo doesn't support building without the `-sb` switch.

Reply:
Fixed in https://github.com/dotnet/source-build-reference-packages/pull/817/commits/8ab54ea0adfc0dd81813d7d749c06f7eb416dbce